modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -21,6 +21,56 @@ definition {
 		}
 	}
 
+	@description = "LPS-161924 - Verify that it's possible to add attachments to the notification template"
+	@priority = "3"
+	test CanAddAttachment {
+		task ("Creating a object and a attachment field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161924",
+				objectName = "CustomObject161924",
+				pluralLabelName = "Custom Objects 161924");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 161924");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldAttachment = "Upload Directly from the User's Computer",
+				fieldLabel = "Custom Attachment",
+				fieldType = "Attachment");
+		}
```

Comment:
Does ObjectAdmin.addObjectFieldViaAPI not work with Attachments?

Reply:
Yes it works, here is an example: `ObjectAdmin.addObjectFieldViaAPI(
					fieldBusinessType = "Attachment",
					fieldLabelName = "From Computer And Show Files In DM",
					fieldName = "fromComputerAndShowFilesInDM",
					fieldType = "Long",
					fileSource = "userComputer",
					isRequired = "false",
					objectName = "UploadObject",
					showFilesInDocumentsAndMedia = "true");`

I'll help @JhossephG with this and check if works for the test

Reply:
No, it doesn't work